libdevcore/Log.h

```diff
@@ -214,27 +214,6 @@ inline boost::log::formatting_ostream& operator<<(
     _strm.stream() << EthCyan "#" << _value.abridged() << EthReset;
     return _strm;
 }
-inline boost::log::formatting_ostream& operator<<(
-    boost::log::formatting_ostream& _strm, h256& _value)
-{
-    auto const& constValue = _value;
-    _strm << constValue;
-    return _strm;
-}
-
-inline boost::log::formatting_ostream& operator<<(
```

Comment:
Removing these probably will result in generic `FixedHash` function used which outputs it with `#` instead of `##`

Reply:
Originally, I thought the `##` was a bug. That's why this change.

Reply:
Why do you still remove these? it seems useful to see that `##38db89c6…` and `#bfd15dbb…` are actually different types of hashes

Reply:
They have different color, isn't this enough?